client/src/services/auth.ts

```diff
@@ -92,6 +94,15 @@ export const signIn = async (
       .auth()
       .signInWithEmailAndPassword(email, password)
       .then(async () => {
+        const user = firebase.auth().currentUser;
+
+        if(user && !user.emailVerified){
+          user?.sendEmailVerification();
+          return {
+            email: "unconfirmed-email",
```

Comment:
We'll want to use AuthErrorMessage["unconfirmed-email"] instead :)

Reply:
gotchu